src/Service/NotificationChannelFactory.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace App\Service;
```

Comment:
It's a bad practice to use Service as a part of the namespace, because in most cases your application has too many services, it's better to organize them by responsibility. E.g. instead of App\Service\NotificationChannelFactory - App\Notification\NotificationChannelFactory